frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -295,6 +326,35 @@ export default function SettingsCluster() {
                 </>
               ),
             },
+            {
+              name: t('translation|Drop Node Shell Linux Image'),
```

Comment:
I think "Drop shell" will be difficult to translate to other languages. But we may ignore the "drop" in those languages I guess.

Reply:
This has been addressed. Resolving.